dev/t3c/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+FROM alpine:latest AS traffic-server-builder
+
+RUN apk add --no-cache build-base perl libexecinfo-dev pcre-dev libressl-dev libtool linux-headers openssl-dev zlib-dev
+ADD https://downloads.apache.org/trafficserver/trafficserver-9.1.0.tar.bz2 /
+RUN tar -xf trafficserver-9.1.0.tar.bz2 && cd trafficserver-9.1.0 && mkdir /ats && ./configure --prefix / --enable-experimental-plugins && make && make install && ls -R /ats
```

Comment:
Using `make -j6` instead of `make` on this line made this layer complete 8m53s faster (5m11s with `-j6` vs 14m4s without)

Reply:
`-j6` can actually cause it to slow down on CPUs with fewer than 6 cores; I'll use `-j` with no argument to allow `make` to select the appropriate DoM for the host CPU.

Reply:
Maybe the documentation should be changed from `make -j4` to just `make -j`:
https://github.com/apache/trafficcontrol/blob/85f1073154a8cf3754fb454984b1e50a7a4e636c/docs/source/admin/quick_howto/ciab.rst#L43

Reply:
Yeah, probably